lib/kafkalib/partition/settings.go

```diff
@@ -43,7 +44,7 @@ func (b *BigQuerySettings) GenerateMergeString(values []string) (string, error)
 	case "time":
 		switch b.PartitionBy {
 		case "daily":
-			return fmt.Sprintf(`DATE(c.%s) IN (%s)`, b.PartitionField, array.StringsJoinAddSingleQuotes(values)), nil
+			return fmt.Sprintf(`DATE(%s.%s) IN (%s)`, constants.TargetAlias, b.PartitionField, array.StringsJoinAddSingleQuotes(values)), nil
```

Comment:
Sneaky!

Reply:
Is `b.PartitionField` always a column? Should we be quoting it?

Reply:
If so could do `QuoteTableAliasColumns(constants.TargetAlias, columns.Column(b.PartitionField, typing.Invalid))`

Reply:
I think using the `sql` package at all may cause an import cycle, so maybe something to look at after this PR.

Reply:
Yes, we should escape it

Reply:
I'll leave a TODO

Reply:
Actually, I can just move this to the bq pkg